build.gradle.kts

```diff
@@ -174,14 +175,22 @@ tasks {
                 "RedundantOverride",
                 "RedundantThrows",
                 "RemoveUnusedImports",
+                "DefaultCharset",
                 "UnnecessarilyFullyQualified",
                 "UnnecessarilyUsedValue",
                 "UnnecessaryBoxedAssignment",
                 "UnnecessaryBoxedVariable",
                 "UnnecessaryFinal",
                 "UnusedException",
+                "UnusedLabel",
+                "UnusedMethod",
+                "UnusedNestedClass",
+                "UnusedVariable",
                 "WildcardImport",
             )
+            if (DefaultNativePlatform.getCurrentOperatingSystem().isWindows) {
+                disable("MisleadingEscapedSpace") // good stuff
```

Comment:
What's so good about this stuff? 😄

Reply:
It only fails on Windows, so I had to disable it in order to compile code 😅

Reply:
It's supposed to check that you only use \s at the end of the line in multiline strings (to preserve whitespaces). Works well on Linux and MacOS.